src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -1,59 +1,56 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{80721425-68E5-48DC-87EA-41D63D0B45FA}</ProjectGuid>
+    <ProjectGuid>{74165807-676F-4464-B607-8770C13E8991}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>FakeItEasy</RootNamespace>
     <AssemblyName>FakeItEasy</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
> Because the test project FakeItEasy.Tests targets v4.5 and it reference the v4.0 project, it get confused by the GetTypeInfo() extension method that I defined for v4.0 for compatibility, and the one from the .NET framework.

I had a similar problem before with one of our extension methods I forget which right now, and I think I ended up moving the extension method to a different namespace…

Oh. I guess this means we're testing the 4.5 version of the library right now, not the 4.0 one? I never looked.